lib/plugins/inspec-parallel/lib/inspec-parallel/command.rb

```diff
@@ -59,6 +60,30 @@ def read_options_file
         opts
       end
 
+      def content_from_file(option_file)
+        if File.exist?(option_file)
+          unless [".sh", ".csh", ".ps1"].include? File.extname(option_file)
+            File.readlines(option_file)
+          else
+            if Inspec.locally_windows? && (File.extname(option_file) == ".ps1")
+              output = `powershell -File "#{option_file}"`
+              output.split("\n")
+            elsif [".sh", ".csh"].include? File.extname(option_file)
+              if system("bash #{option_file}", out: File::NULL)
+                output = `bash "#{option_file}"`
```

Comment:
I'm concerned here that we're executing the file twice. Is there anyway we could keep the stdout from the first execution, and if it worked, use it?

Reply:
Let me check a workaround for this.